src/Apps/Consign/consignFromCollectorProfileMyCollectionRoutes.tsx

```diff
@@ -127,6 +127,7 @@ export const consignFromCollectorProfileMyCollectionRoutes: AppRouteConfig[] = [
     children: [
       {
         path: "/",
+        layout: "FullBleed",
```

Comment:
I don't really understand why we have the `/sell` route three times. @dariakoko, do you remember why we need this file? I found this [PR](https://github.com/artsy/force/pull/11561) that introduced the new routes but couldn't find enough info in the ticket on why it's necessary. Do you think we could remove this duplication? It took me a while to figure out why my layout change was not applied and that this route exist's a couple of times.

Reply:
It's very confusing... I will look into this
Maybe it had something to do with changing the route names

Reply:
I think consignFromMyCollectionRoutes can be deleted and consignFromCollectorProfileMyCollectionRoutes should be refactored. 

We should keep consignFromCollectorProfileMyCollectionRoutes (and rename it...) because here we have the routes for when the user submits an artwork from my collection. 
/sell route should be removed from consignFromCollectorProfileMyCollectionRoutes imo
I opened a draft PR as a followup: [13836](https://github.com/artsy/force/pull/13836). Thank you for raising it!